### PR TITLE
TOPdesk Connector Fixed after Feedback

### PR DESCRIPTION
**Pull request by CJusjong**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@CJusjong Please fix below errors to get your swagger validated for review


`##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1operators~1id~1{id}~1operatorgroups/delete/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1operators~1id~1{id}~1permissiongroups/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1operators~1id~1{id}~1permissiongroups/post/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1operators~1id~1{id}~1permissiongroups/delete/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1operators~1id~1{id}~1unarchive/patch/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons/get/parameters/2
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons/get/parameters/3
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons~1free_fields~1{tab}~1searchlists~1{searchlist}/get/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons~1free_fields~1{tab}~1searchlists~1{searchlist}/get/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons~1free_fields~1{tab}~1searchlists~1{searchlist}/get/parameters/2
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons~1free_fields~1{tab}~1searchlists~1{searchlist}/get/parameters/3
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons~1free_fields~1{tab}~1searchlists~1{searchlist}/post/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons~1free_fields~1{tab}~1searchlists~1{searchlist}/post/parameters/1
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons~1id~1{id}/patch/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons~1id~1{id}~1archive/patch/parameters/0
##[error]The 'x-ms-summary' property is required. ValuePath: /paths/~1persons~1id~1{id}~1unarchive/patch/parameters/0
##[error]The connector category is invalid. Available categories are: [AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website]. ValuePath: /x-ms-connector-metadata/2/propertyValue`
##[error]The maximum allowed number of categories for the connector is '2'. ValuePath: /x-ms-connector-metadata/2/propertyValue

File: certified-connectors/LMS365/apiDefinition.swagger.json [hunk 1]
```diff
@@ -1557,6 +1557,64 @@
                   },
                   "description": "Trainers"
                 },
+                "Admins": {
```
Comment (sriyen-msft) [inline certified-connectors/LMS365/apiDefinition.swagger.json:1560]: @CJusjong Please create a new PR for `LMS365` as it is difficult to review multiple connectors in a single PR and the other connector have few issues need to be addressed. We can not proceed for production deployment for future updates of `LMS365` connector as the existing update is still not merged.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@CJusjong I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by CJusjong** [comment]
Hi Thank You very much. We are working, at solving the issues, there are some repeating IDs, that needs to be handled.

Venlig Hilsen / Best Regards

Christian Kjær Jusjong
Product Specialist
[TOPdesk]
Borupvang 3
2750 Ballerup
+45 2523 7458
+45 7015 7500<tel:+4570157500>
www.topdesk.dk<http://www.topdesk.dk/>

***@***.***<https://blog.topdesk.com/dk>

**Comment by sriyen-msft** [comment]
@CJusjong I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by sriyen-msft** [comment]
@CJusjong I am closing this PR as there is no response and you, you can reopen the same PR whenever you are ready for certification. Thanks

**Pull request closed without merging**